tests/acceptance/features/bootstrap/WebUIGuestsContext.php

```diff
@@ -91,12 +121,59 @@ public function guestUserRegistersUsingWebUI($guestDisplayName, $password) {
 		$this->featureContext->rememberUserPassword($userName, $password);
 	}
 
+	/**
+	 * @When the user shares file :fileName with guest user with email :email using webUI
+	 *
+	 * @param string $fileName
+	 * @param string $email
+	 *
+	 * @return void
+	 */
+	public function theUserSharesFileWithGuestUserWithEmailUsingWebui($fileName, $email) {
+		$this->filesPage->openSharingDialog($fileName, $this->getSession());
+		$sharingDialog = $this->filesPage->getSharingDialog();
+		$userAddDialog = sprintf($this->userAddDialogBoxFramework,$email);
+		$sharingDialog->shareWithUserOrGroup(
+			$email, $userAddDialog, $this->getSession()
+		);
+		$this->featureContext->addUserToCreatedUsersList($email, null);
+	}
+
+	/**
+	 * @Then the user should not be able to share the file with guest user :user
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userShouldNotBeDisplayedInTheDropdownAsGuestUser($user) {
+		$sharingDialogue = $this->filesPage->getSharingDialog();
```

Comment:
Would be good to be consistent with `Dialogue`/`Dialog`.
```suggestion
		$sharingDialog = $this->filesPage->getSharingDialog();
```